app/src/main/res/layout/fragment_updatebalance.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.UpdateBalanceFragment">
+
+    <TextView
+        android:id="@+id/text_accountName"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Account to be Updated"
+        android:textAlignment="center"
+        android:textSize="20sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Spinner
+        android:id="@+id/spinnerAccountType"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:entries="@array/account_arrays"
```

Comment:
We'll need to take care of this dynamically rather than with a list declared as a string resource. I'll take care of it, but you can remove this `android:entries` row and the string values. It'll be only based on the inputted accounts the user has declared on the other screen.

Reply:
removed